CONTRIBUTING/index.html

```diff
@@ -1,4168 +1,68 @@
-<h3 id="committing">Committing<a class="headerlink" href="#committing" title="Permanent link">&para;</a></h3>
-<p>See the <a href="../running-locally/#committing">Committing Guidelines</a>.</p>
-<h3 id="dependencies">Dependencies<a class="headerlink" href="#dependencies" title="Permanent link">&para;</a></h3>
-<p>Dependencies increase the risk of security issues and have on-going maintenance costs.</p>
-<p>The dependency must pass these test:</p>
-<ul>
-<li>A strong use case.</li>
-<li>It has an acceptable license (e.g. MIT).</li>
-<li>It is actively maintained.</li>
-<li>It has no security issues.</li>
-</ul>
-<p>Example, should we add <code>fasttemplate</code>
-, <a href="https://snyk.io/advisor/golang/github.com/valyala/fasttemplate">view the Snyk report</a>:</p>
-<table>
-<thead>
-<tr>
-<th>Test</th>
-<th>Outcome</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>A strong use case.</td>
-<td>❌ Fail. We can use <code>text/template</code>.</td>
-</tr>
-<tr>
-<td>It has an acceptable license (e.g. MIT)</td>
-<td>✅ Pass. MIT license.</td>
-</tr>
-<tr>
-<td>It is actively maintained.</td>
-<td>❌ Fail. Project is inactive.</td>
-</tr>
-<tr>
-<td>It has no security issues.</td>
-<td>✅ Pass. No known security issues.</td>
-</tr>
-</tbody>
-</table>
-<p>No, we should not add that dependency.</p>
-<h3 id="test-policy">Test Policy<a class="headerlink" href="#test-policy" title="Permanent link">&para;</a></h3>
-<p>Changes without either unit or e2e tests are unlikely to be accepted.
-See <a href="https://github.com/argoproj/argo-workflows/blob/main/.github/pull_request_template.md">the pull request template</a>.</p>
-<h3 id="contributor-workshop">Contributor Workshop<a class="headerlink" href="#contributor-workshop" title="Permanent link">&para;</a></h3>
-<p>Please check out the following resources if you are interested in contributing:</p>
-<ul>
-<li><a href="https://youtu.be/zZv0lNCDG9w">90m hands-on contributor workshop</a>.</li>
-<li><a href="https://docs.google.com/presentation/d/1IU0a3unnr3tBRi38Zn3EHQZj3z6yvocfG9x9icRu1LE/edit?usp=sharing">Deep-dive into components and hands-on experiments</a>.</li>
-<li><a href="https://github.com/argoproj/argo-workflows/blob/main/docs/architecture.md">Architecture overview</a>.</li>
-</ul>
-
-              
-
-<!-- Giscus -->
-<h2 id="__comments">Comments</h2>
-
-<script src="https://giscus.app/client.js"
-        data-repo="argoproj/argo-workflows"
-        data-repo-id="MDEwOlJlcG9zaXRvcnkxMDA5ODI0NDk="
-        data-category="General"
-        data-category-id="MDE4OkRpc2N1c3Npb25DYXRlZ29yeTMyMTA3OTY1"
-        data-mapping="pathname"
-        data-reactions-enabled="1"
-        data-emit-metadata="0"
-        data-input-position="bottom"
-        data-theme="light"
-        data-lang="en"
-        data-loading="lazy"
-        crossorigin="anonymous"
-        async>
-</script>
 
+              <h1>Contributing - Argo Workflows - The workflow engine for Kubernetes</h1>
```

Comment:
```suggestion
              <h1>Contributing</h1>
```

There is a small issue here and in [the script](https://gist.github.com/jmeridth/94cd15d2b11d444fdf91913aa59f34ab#file-update-aw-github-pages-py-L56). The heading should only have the page title and not the full window title (as in the `<title>` element).

That's why I had in [my template](https://github.com/argoproj/argo-workflows/pull/12362#discussion_r1433264020):
```html
<title>{{page_title}} - Argo Workflows - The workflow engine for Kubernetes</title>
```
```html
<h1>{{page_title}}</h1>
```

I'm guessing that you changed that because there are a few odd ones out; `index.html` and `404.html` just say "Argo Workflows - The Workflow engine for Kubernetes" for the title and "Argo Workflows" for the heading.
So we're probably going to want to special case those in the script + template

Reply:
Done.  Also had to handle the base index.html and base 404.html.  Went with the `Argo Workflows - The workflow engine for Kubernetes` text for both page title and page H1.

Also added comment to python script giving you credit for the HTML template.

Reply:
Yea that's what I mentioned above -- if you want to match the existing headings for both, it's:
- ["Argo Workflows"](https://github.com/argoproj/argo-workflows/pull/12362#discussion_r1435888670) for base `index.html`
- ["404 - Not found"](https://github.com/argoproj/argo-workflows/pull/12362#discussion_r1435888503) for `404.html`

Title was "Argo Workflows - The Workflow engine for Kubernetes" [for](https://github.com/argoproj/argo-workflows/pull/12362/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L16) [both](https://github.com/argoproj/argo-workflows/pull/12362/files#diff-128cc35baaf40bac8ae4d2932ca6e009311e8279f03f258e9cc83966f1d5b32eL16)